src/Tribute.js

```diff
@@ -349,7 +359,7 @@ class Tribute {
 
     selectItemAtIndex(index, originalEvent) {
         index = parseInt(index)
-        if (typeof index !== 'number') return
+        if (typeof index !== 'number' || isNaN(index)) return
```

Comment:
bcoz: if was facing problem when index was undefined, parseInt(undefined) becomes NaN and 
(typeof Nan) evaluates to number